docs/actions-cohortextractor.md

```diff
@@ -56,13 +56,15 @@ A basic `generate_cohort` action looks like this:
 
 ```yaml
 generate_study_cohort
-  run: cohortextractor:latest generate_cohort --output-format=csv.gz
+  run: cohortextractor:latest generate_cohort --output-format =csv.gz
   outputs:
     highly_sensitive:
       data: output/input.csv.gz
+    ### This file is produced in compressed format. If using Stata for your analysis you may need to remove 
+    ### the --output-format option and the .gz file extension to produce an uncompressed CSV instead.
```

Comment:
Would it be clearer to say "use `--output-format=csv`", rather than to say "remove the option"? No particular strong feeling, but it seems a bit more consistent to me.

Reply:
OK done